### DISCUSSION TITLE
Import Report Greenbone (Openvas) XML

### DISCUSSION THREAD

**Discussion by c0dr123** [DefectDojo/django-DefectDojo#8750] [Ideas]
Dear Sir or Madam,

we use the vendor Greenbone, the company distributes the paid version of Openvas. The paid version allows only export to XML 
 / json format of a report. Could you enable an import from Greenbone (Openvas) to XML / json file?

**Comment by manuel-sommer** [comment]
Hi @c0dr123,

so, a csv output for this parser is not supported: https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/tools/openvas_csv/parser.py ?
Then, I recommend to open up an issue including an annonymized output file. Then, it is possible to write a parser.

**Reply by c0dr123** [reply]
Hello Manuel-Sommer,
thank you for the quick feedback. No Greenbone does not offer an export for CSV for the paid version of Openvas. I have been in contact with Greenbone support on this issue. I will create an issue ticket from this discussion.

**Comment by c0dr123** [comment]
>

> Hi @c0dr123,
> 
> so, a csv output for this parser is not supported: https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/tools/openvas_csv/parser.py ? Then, I recommend to open up an issue including an annonymized output file. Then, it is possible to write a parser.
